src/components/ProgressBar/index.tsx

```diff
@@ -17,15 +18,15 @@ const shineAnimation = keyframes`
 export const progressBarVariants = ['primary', 'success', 'warning', 'info']
 
 const StyledBox = styled(Box, {
-  shouldForwardProp: prop => !['backgroundColor'].includes(prop),
-})`
+  shouldForwardProp: prop => !['backgroundColor'].includes(prop.toString()),
```

Comment:
simpler & efficient (but less extensible) suggestion:
```suggestion
  shouldForwardProp: prop => 'backgroundColor' === prop,
```

Reply:
This is the convention we have elsewhere, we can keep it that way to be consistent :)